spark/src/main/scala/ai/chronon/spark/Analyzer.scala

```diff
@@ -186,11 +186,15 @@ class Analyzer(tableUtils: TableUtils,
   def analyzeGroupBy(groupByConf: api.GroupBy,
                      prefix: String = "",
                      includeOutputTableName: Boolean = false,
-                     enableHitter: Boolean = false): (Array[AggregationMetadata], Map[String, DataType]) = {
+                     enableHitter: Boolean = false,
+                     validationAssert: Boolean = false): (Array[AggregationMetadata], Map[String, DataType]) = {
```

Comment:
we want to perform sanity check for time column for all the cases. So we can remove this flag. 
As long as there is time column, then we should check it.

Reply:
Done.